locale/action_names.rb

```diff
@@ -0,0 +1,7 @@
+# Autogenerated!
+_("Action with sub plans")
+_("Create architecture")
+_("Delete architecture")
+_("Update architecture")
+_("Import facts")
+_(" on ")
```

Comment:
@stbenjam [foreman_remote_execution/pull/153](https://github.com/theforeman/foreman_remote_execution/pull/153) should fix this issue by renaming the action to `Remote action` and using `humanized_input` for displaying the rest